framework/content/ecl-typography/ecl-typography-paragraphs/_paragraphs.scss

```diff
@@ -1,35 +1,26 @@
-.paragraph {
-  display: block;
+.ecl-p {
```

Comment:
Same remark as for `.ecl-h`, what does `p` mean? TBD internally how we should name those classes